### ISSUE TITLE
Issue on docs

### ISSUE THREAD

**Issue by arfinraees7-pixel** [chainstack/dev-portal#130]
Path: /docs/solana-creating-a-def simple_strategy(prices):
    short_ma = sum(prices[-5:]) / 5
    long_ma = sum(prices[-20:]) / 20
    if short_ma > long_ma:
        return 'BUY'
    elif short_ma < long_ma:
        return 'SELL'
    return 'HOLD'
pumpfun-botimport requests

def get_price(token_mint="So11111111111111111111111111111111111111112", quote_mint="USDC"):
    url = f"https://quote-api.jup.ag/v6/quote?inputMint={token_mint}&outputMint={quote_mint}&amount=1000000&slippage=1"
    response = requests.get(url)
    data = response.json()
    return float(data['data'][0]['outAmount']) / 1e6  # Adjust for decimals
pip install solana
from solana.rpc.api import Client
from solana.keypair import Keypair
from solana.transaction import Transaction

solana_client = Client("https://api.mainnet-beta.solana.com")
wallet = Keypair.from_secret_key(bytes(...))  # Load from file or env
import time

price_history = []

while True:
    try:
        price = get_price()
        price_history.append(price)
        if len(price_history) > 20:
            signal = simple_strategy(price_history)
            print(f"[Signal] {signal} at {price}")
            if signal == 'BUY':
                # execute_swap_buy()
                pass
            elif signal == 'SELL':
                # execute_swap_sell()
                pass
        time.sleep(60)  # Run every minute
    except Exception as e:
        print("Error:", e)
        time.sleep(60)
from sklearn.linear_model import LogisticRegression
import numpy as np

# Example dataset
X = np.array([...])  # Features like RSI, MACD, MA
y = np.array([...])  # Labels: 1 for BUY, 0 for SELL

model = LogisticRegression().fit(X, y)
prediction = model.predict(latest_features)

**Comment by arfinraees7-pixel** [comment]
Bot connect phantom wallet
